backend/app/store/engine/bolt.go

```diff
@@ -617,6 +646,140 @@ func (b *BoltDB) flagBucket(tx *bolt.Tx, flag Flag) (bkt *bolt.Bucket, err error
 	return bkt, nil
 }
 
+// getUserDetail returns UserDetailEntry with requested userDetail (omitting other details)
+func (b *BoltDB) getUserDetail(req UserDetailRequest) (result []UserDetailEntry, err error) {
```

Comment:
To provide a single interface both for getting single UserDetail and listing all user details for the whole site.

Previously it was two different functions with two signatures (`UserDetail` and `ListUserDetails`) but if I understood you correctly you were against extending the `engine` interface without absolute necessity and I've [merged them](https://github.com/umputun/remark/pull/469/commits/05fed5730ebe464cb0cf9f20e1441ee90831a602#diff-ce8df428a3e2494038b311335b9e8c95L29). As it's a list now, `UserDetailEntry` started having `UserID` duplicated in to prevent returning map (by  `UserID`) instead of list for `listUserDetails`.

The previous version of `UserDetail` returned just `string` (as it's a result for a single entry) and if we want to split single detail handling and list again new one would be returning single `UserDetailEntry`. Please let me know if that's necessary.

Reply:
> To provide a single interface both for getting single UserDetail and listing all user details for the whole site.

It makes sense, just not as obvious as I read this particular method. Probably adding a comment explaining why list returned will be enough

Reply:
Addressed by eccc90596c53cc0da95af5ffb47c17a8d7d2cb55, please let me know if that's clear enough.

Reply:
LGTM